### DISCUSSION TITLE
Missing var A320NX ....

### DISCUSSION THREAD

**Discussion by Jeanlouis1212** [flybywiresim/aircraft#7991] [Help] [answered]
J'arrive just sur votre site .  J'espére y trouver  de l'aide .
Meilleures salutations.
NB : mon anglais est nul ; les toutils de traduction sont meilleurs !!!

J'utilise W10 ,FS2020, A32NX et les  appareils de FSCockpit .
J'utilise le driver de FSCockpit  v.1.6.4.0 for Fly by wire A32NX .
et voila le résultat :
![852147](https://user-images.githubusercontent.com/132000600/234941029-87284173-9a3d-425d-8abe-db6d8a1de80d.PNG)
Merci de vos conseils .
Jean louis

**Comment by Muyz** [comment, marked as answer]
This is an FSCockpit issue. Please update to the latest version of FSCockpit. Please contact me or Tomasz from FSCockpit. Thanks
